src/Microsoft.DotNet.Build.Tasks/GenerateEncodingTable.cs

```diff
@@ -0,0 +1,395 @@
+﻿using Microsoft.Build.Framework;
```

Comment:
Excellent! 
I assume there will be no change in the corefx side except updating the new generated cs encoding data file. right?